src/pages/user/booking/index.tsx

```diff
@@ -161,7 +161,7 @@ const TrainerFilter = () => {
           {trainerProfiles.map((trainerProfile, idx) => {
             return (
               <div key={idx}>
-                <Link href={"/user/booking/trainer/" + trainerProfile.username}>
+                <Link href={"/user/booking/" + trainerProfile.username}>
```

Comment:
to be safe from injection, i think we should start doing this properly now (https://nextjs.org/docs/routing/introduction#linking-to-dynamic-paths). we should always use encoded url

Reply:
also, i think there are many part that we (including me) do this wrong, so if anyone see non encoded dynamic url, they should fix.